coral/src/domain/topic/index.ts

```diff
@@ -23,7 +23,7 @@ import {
   TopicSchemaOverview,
   TopicTeam,
 } from "src/domain/topic/topic-types";
-
+import { PromotionStatus } from "src/domain/promotion/promotion-types";
```

Comment:
Sorry, I should have defined that more in detail in my last comment. 

You could delete the whole `/domain/promotion` directory. And instead of defining `PromotionStatus` there, defining it as `TopicPromotionStatus` in `topic-types.ts` and then import / export in from the index file here like you did. 

The idea behind the `/domain` directory is to have a place where we kind of have our "domain model definitions". Since `PromotionStatus` was more general, it had it's own directory `promotion`. Now that we have `TopicPromotionStatus`, this is only related to topic and should be part of `domain/topic` only. 

BUT you can also leave it as it is for now & I'll move it when I do the feature.